tests/migration/version_switch_upgrade_target.pm

```diff
@@ -47,6 +47,7 @@ sub run {
 
     record_info('Version', 'VERSION=' . get_var('VERSION'));
     reconnect_mgmt_console if is_pvm;
+    reset_consoles_tty;
```

Comment:
if is_pvm then
reconnect_mgmt_console 
else
reset_consoles_tty

Reply:
I checked powerVM, Don't need this condition, it still can work.